src/workflow.scss

```
@@ -930,6 +930,128 @@ $font-sans: 'Montserrat', sans-serif !important;
   opacity: .25 !important;
 }
 
+/* CONTENT INTEGRATION */
+workflow-extension-highlight {
+  display: inline-block !important;
+  background-color:#022242 !important;
+  color:#fff !important;
+  font-weight: 700 !important;
+  cursor: pointer !important;
+  position: relative !important;
+  transition: 0.25s !important;
+}
+
+workflow-extension-highlight .workflow-extension-highlight-tooltip {
+  margin: 0px !important;
+  padding: 0px !important;
+  width: 300px !important;
+  height: auto !important;
+
+  position: absolute !important;
+  left: 50% !important;
+  top: calc(100% + 6px) !important;
+  transform: translate(-50%) !important;
+
+  box-sizing: border-box !important;
+  border: none !important;
+  border-radius: 20px !important;
+  
+  background-color: #022242;
+  font-size: 14px !important;
+  line-height: 1.3 !important;
+  font-family: $font-sans;
+  font-style: normal !important;
+  color: #fff !important;
+  cursor: initial !important;
+  -webkit-font-smoothing: antialiased !important;
+
+  opacity: 0 !important;
+  visibility: hidden !important;
+  transition: 0.25s !important;
+  z-index: 99999999998 !important;
+}
+
+workflow-extension-highlight:hover .workflow-extension-highlight-tooltip {
+  opacity: 1 !important;
+  visibility: visible !important;
+}
+
+workflow-extension-highlight .workflow-extension-highlight-tooltip-header {
+  padding: 12px !important;
+  text-align: center !important;
+  z-index: 99999999999 !important;
+}
+
+workflow-extension-highlight .workflow-extension-highlight-tooltip-header span{
+  text-transform: uppercase !important;
+}
+
+workflow-extension-highlight .workflow-extension-highlight-tooltip-publisher-content,
+workflow-extension-highlight .workflow-extension-highlight-tooltip-by {
+  margin: 0 !important;
+  padding: 0 !important;
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+}
+
+workflow-extension-highlight .workflow-extension-highlight-tooltip-publisher-content {
+  font-size: 18px !important;
+}
+
+workflow-extension-highlight .workflow-extension-highlight-tooltip-publisher-content span {
+  display: flex !important;
+}
+
+workflow-extension-highlight .workflow-extension-highlight-tooltip-publisher-content svg {
+  margin-right: 8px !important;
+}
```

This is scss, so nesting is supported and helps mimic some html structure for readability too.

```suggestion
workflow-extension-highlight .workflow-extension-highlight-tooltip-publisher-content {
  font-size: 18px !important;
  
  span {
  	display: flex !important;
  }
  
  svg {
  	margin-right: 8px !important;
  }
}
```